controllers/articles.controller.js

```diff
@@ -49,4 +50,22 @@ const patchArticleVotes = (req, res, next) => {
     .catch(next);
 };
 
-module.exports = { getArticleById, getArticlesByLifo, patchArticleVotes };
+const getArticlesByQuery = (req, res, next) => {
+  const query = req.query.topic;
+
```

Comment:
I think you should probably check here if the topic is valid (using your getTopics model!)